stages/2-provider-infra-setup/infra-setup

```diff
@@ -10,7 +10,7 @@ cp  .kube/config ~/.kube/config
 
 kubectl get po
 
```

Comment:
dont edit this variable as it will break other things. just create a new variable for url without port and use that to create configmap

Reply:
use this variable director_url